config/changelogger.php

```diff
@@ -13,4 +13,16 @@
     */
     'directory'  => env('DIRECTORY', getcwd()),
     'unreleased' => env('DIRECTORY', getcwd()) . '/changelogs/unreleased',
+    'types'      => [
```

Comment:
Use `env()` so others can change the types to fit their needs

Reply:
Enhance feature to read custom types from `.changelogger.yml` or if no custom types are provided fallback to default types.